addons/account_add_gln/models/res_partner.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    account_gln = fields.Char(string="GLN", help="Global Location Number")
```

Comment:
```suggestion
    global_location_number = fields.Char(string="GLN", help="Global Location Number")
```
AFAIK this identifier is not really related to Accounting so maybe name it like this ? 🤔